site/app/models/Gradeable.php

```diff
@@ -560,6 +560,14 @@ public function getVersions() {
     }
 
     /**
+     * Returns the total number of points possible for testcases
+     * @return float
+    */
+    public function getTotalNonExtracreditAutograderPoints() {
```

Comment:
This matches the function definition of `getTotalAutograderNonExtraCreditPoints()` so this is unnecessary.